packages/api/src/modules/stats/repositories/statsRepository.test.ts

```diff
@@ -0,0 +1,37 @@
+import statsRepository from "./statsRepository"
+
+const TODAY = new Date(Date.UTC(2022,0,1));
+
+const dateFactory = (diff: number) => { 
+    const date = new Date(TODAY);
+    date.setDate(TODAY.getDate() + diff);
+    return date;
+};
+
+describe("StatsRepository", () => {
+    describe("countUsersByRequestNbOnPeriod()", () => {
+        const NB_REQUESTS = 3;
+        const START = dateFactory(-1);
+        const END = dateFactory(1);
+
+        it('should be call mongo with admin filters', async () => {
```

Comment:
"should call", car "should be call" ne veut pas dire la même chose ("devrait être appelé par mongo" vs "doit appeler mongo"). Il y a cette erreur un peu partout. :)

Reply:
Done